sdk/python/feast/permissions/auth/oidc_token_parser.py

```diff
@@ -56,6 +57,17 @@ async def user_details_from_access_token(self, access_token: str) -> User:
             AuthenticationError if any error happens.
         """
 
+        skip_auth_key = os.getenv("SKIP_AUTH_KEY")
+        if skip_auth_key:
+            try:
+                decoded_token = jwt.decode(access_token)
```

Comment:
if `skip_auth_key` it's true why it's decoding the token ? in this case which user token it will use ?

Reply:
This flag is with a text, it is decoding the client token value to make sure it is the same value as the env var the server has